qa/tasks/cephfs/test_admin.py

```diff
@@ -2191,6 +2191,16 @@ class TestFSFail(TestAdminCommands):
     MDSS_REQUIRED = 2
     CLIENTS_REQUIRED = 1
 
+    def tearDown(self):
```

Comment:
Why override tearDown()? Would it not suffice to bring the file system online or the MDS back active in those tests?

Reply:
This was done to avoid code duplication (these 3 lines would become 24 lines) and also because these lines are part of teardown, actual test has nothing to do with it..

Reply:
The actual tests does have everything to do with it since those tests are failing the file system and/or the MDSs.

Reply:
Okay, I'll delete tearDown and copy these lines in each test...

Reply:
Done.